curriculum/challenges/english/25-front-end-development/lecture-working-with-file-systems/672ac3c5d0e9fd31835ff772.md

```diff
@@ -10,6 +10,74 @@ dashedName: how-to-create-move-and-delete-files-and-folders-using-explorer-finde
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+Hoe to create, move, and delete files and folders using the Explorer or Finder?
+
+To create a file on Windows using the Explorer, you can click on the "New" option at the top right. This will show you a list of the different types of files that you can create. 
+
+You can also create files in their corresponding applications. Let's create a text file to show you the process.
+
+When you click on "New" and select the file type, you will see a new text file in your current folder with a default name. You can rename it if you'd like.
+
+The process of creating folders is exactly the same. Click the "New" button at the top right and then select the "Folder" option or right-click on an empty spot and go to "New." Then, select the "Folder" option. You'll see a new empty folder in your current location.
+
+You can rename files and folders by selecting them and clicking on the "Rename" option at the top. When current name of the file is highlighted, you can start writing the new name of the file. Press Enter to confirm the changes.
+
+Alternatively, you can right-click on the file or folder and select the "Rename" option or use the F2 keyboard shortcut. Write the new name and press Enter to confirm the changes. 
+
+You can move a file or folder to another folder by dragging and dropping it into the destination folder, if you have it opened it as a tab. 
+
+You can also select the file or folder and select the "Cut" option at the top.
+
+Then, go to the destination folder and paste it by clicking the "Paste" button at the top. If you copied or cut the file previously, this button will be enabled.
+
+You can also do this by right-clicking on an empty spot and then clicking on the paste icon. This will remove the file or folder from its original location.
+
+To delete a file or folder, click on it and then click on the trash icon at the top. You will also find a "Delete" option if you right-click on it.
+
+Great. Now let's see how you can do this on macOS using Finder. 
+
+Finder is the default file manager on macOS.
+
+To create a file on macOS, you need to open an application that lets you create the type of document that you need to create. Currently, there isn't a built-in way to create a file directly on Finder.
+
+For example, to create a simple plain text file, you can open TextEdit and save your file. I will use text files to show you the process but this works exactly the same for any type of file.
```

Comment:
The usage of first form "I will" doesn't fit here. 

Maybe this could be changed to:

```suggestion
For example, to create a simple plain text file, you can open TextEdit and save your file. The process works exactly the same for any type of file.
```